### PR TITLE
Page not Found 404

### PR DESCRIPTION
**Pull request by Vaibhavabhaysharma**
Error : 404 page not found html file with related svg

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+    
+</head>
+
+<body>
+    <section id="wrapper" class="error-page">
+        <div class="error-box">
+            <div class="error-body text-center">
+                <h1>404</h1>
+                <h3 class="text-uppercase">Page Not Found !</h3>
+                <p class="text-muted m-t-30 m-b-30">The page you were looking for does not exist! Click <a href="">here</a> to return to the Home Page.</p>
```
[the remark this answers is not in the document]
Comment (Vaibhavabhaysharma) [reply 404.html]: Requested changes done👍🏻

File: 404.html [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+            font-family: sans-serif;
+            line-height: 1.15;
+            -webkit-text-size-adjust: 100%;
+            -ms-text-size-adjust: 100%;
+            -ms-overflow-style: scrollbar;
+            -webkit-tap-highlight-color: transparent;
+                }
+        body {
+            background: #fff;
+            font-family: "Montserrat", sans-serif;
```
[the remark this answers is not in the document]
Comment (Vaibhavabhaysharma) [reply 404.html:28]: But, its not an inline css. Its  an internal css as I included it in  a <style> tag .

[the remark this answers is not in the document]
Comment (Vaibhavabhaysharma) [reply 404.html:28]: Please explain it

File: not-found.svg [hunk 3]
```diff
@@ -0,0 +1,246 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
```
[the remark this answers is not in the document]
Comment (Vaibhavabhaysharma) [reply not-found.svg:3]: Sir, I don't know how to change code of svg , so please tell me which command is to be added and where?

File: 404.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (smaranjitghose) [reply 404.html:28]: My Bad. I meant do not use internal css please

**Comment by smaranjitghose** [comment]
We usually do not need a footer for the 404 page. can you please remove it. Let the svg part be there. I'll take care of it. Can you just revamp the css as suggested

File: 404.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (Vaibhavabhaysharma) [reply 404.html]: Resolved issue in pr 162

File: not-found.svg (hunk 3 above)
[the remark this answers is not in the document]
Comment (Vaibhavabhaysharma) [reply not-found.svg:3]: Resolved issue in pr 162

**Pull request closed without merging**